examples/typescript-reactflux/README.md

```diff
@@ -0,0 +1,87 @@
+* [ScreenCast - Why I Like TypeScript](https://www.leebrimelow.com/why-i-like-typescripts/)
+
+Get help from other TypeScript users:
+
+* [TypeScript on StackOverflow](https://stackoverflow.com/questions/tagged/typescript/)
+* [Forums](https://typescript.codeplex.com/discussions/)
+* [TypeScript on Twitter](https://twitter.com/typescriptlang/)
+
+_If you have other helpful links to share, or find any of the links above no longer work, please [let us know](https://github.com/tastejs/todomvc/issues/)._
+
+
+## Implementation
+
+This application is a port of the [flux-todomvc](https://github.com/facebook/flux/tree/master/examples/flux-todomvc/) example, which is
+part of Facebook's [Flux](https://github.com/facebook/flux/) project.
+
+The original example uses [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html/), which is a a JavaScript syntax extension that looks similar to XML.
+Transforming JSX to Javascript is pretty easy as James Brantly explained in his recent React.js Conf 2015 talk
+[Static typing with Flow and TypeScript](https://conf.reactjs.com/schedule.html#static-typing-with-flow-and-typescript/).
+
+James Brantly's [reactconf](https://github.com/jbrantly/reactconf/) uses [webpack](https://webpack.github.io/) and [ts-jsx-loader](https://github.com/jbrantly/ts-jsx-loader/).
+This project on the other hand uses [grunt](https://gruntjs.com/) and [grunt-text-replace](https://github.com/yoniholmes/grunt-text-replace/) with
+[react-tools](https://www.npmjs.com/package/react-tools/) and [grunt-browserify](https://github.com/jmreidy/grunt-browserify/) instead.
+
+The main reason for [choosing browserify over webpack](https://blog.namangoel.com/browserify-vs-webpack-js-drama/) is that
+browserify injects node.js polyfills like [EventEmitter](https://nodejs.org/api/events.html#events_class_events_eventemitter/),
+which are used by the original [flux-todomvc](https://github.com/facebook/flux/tree/master/examples/flux-todomvc/) example.
+
+During the build phase all `*.tsx` and `*.ts` are first being copied to a temporary `./build` folder and then compiled to Javascript using the Typescript compiler.
+A second pass then intelligently replaces `React.jsx(...)` with transformed JSX code. For more details see `./grunt/replace.js`.
+The build mechanics are all implemented in `./grunt/aliases.js`. The output folder is `./public`.
+
+This project also supports minification using Google's Closure Compiler.
+
+
+## Running
+
+You must have [node.js](https://nodejs.org/) installed on your computer, which includes [npm](https://www.npmjs.org/). From the root project directory run these commands from the command line:
+
+```
+npm install
+```
+
+That will install all dependencies and build `public/js/bundle.js`, which is used by `public/index.html`.
+Once you've built the project you can open `public/index.html` in your browser, or simply run `npm start`.
```

Comment:
This example runs from `todomvc.com`, or directly in the browser, because I checked in `bundle.min.js`. Those are instructions for devs, who want to copy the example and modify it for their needs. I need to change the folder name from `public` to `web`, though.